src/components/ViewRequest/Actions/ReworkRequest.tsx

```diff
@@ -71,12 +93,9 @@ const ReworkRequest = () => {
         <DialogBody>
           <DialogTitle>Rework request</DialogTitle>
           <DialogContent>
-            <p>
-              Send request back a step?
-              <br />
-              Fill out this modal with a form for sending a request back a step
-              with a reason. (Notes capability pending)
-            </p>
+            <Field label="Rework reason">
+              <Textarea value={reason} onChange={updateReason} />
+            </Field>
```

Comment:
Should this be a required field to send it back? Didn't see it as an acceptance criteia, but thought it was discussed at some point.